docs/topics/custom-images.md

```diff
@@ -0,0 +1,130 @@
+# Custom images
+
+This document will help you get a CAPZ Kubernetes cluster up and running with your custom image.
+
+## Reference images
+
+An *image* defines the operating system and Kubernetes components that will populate the disk of each node in your cluster.
+
+By default, images offered by "capi" in the [Azure Marketplace][azure-marketplace] are used. You can list these *reference images* with this command:
+
+```bash
+az vm image list --publisher cncf-upstream --offer capi --all -o table
+```
+
+But for more control over your nodes, you should build, publish, and use a *custom image*.
```

Comment:
is shared image gallery considered "publish"? if not, let's drop publish from here